conans/model/conf.py

```diff
@@ -49,6 +50,7 @@
     "tools.cmake.cmaketoolchain:toolset_arch": "Toolset architecture to be used as part of CMAKE_GENERATOR_TOOLSET in CMakeToolchain",
     "tools.cmake.cmaketoolchain.presets:max_schema_version": "Generate CMakeUserPreset.json compatible with the supplied schema version",
     "tools.cmake.cmake_layout:build_folder_vars": "Settings and Options that will produce a different build folder and different CMake presets names",
+    "tools.files.download:download_cache": "Define the cache folder to store downloads from files.download()/get()",
```

Comment:
Why do we need two separate confs? Why not using only one configuration and store all the downloaded files in the same cache?

Reply:
There are 2 reasons:
- One technical: the ``conanfile.conf`` received by recipes do not have visibility over the ``core.*`` configuration that is suppossed to be internal only, but not visible or usable by recipes. But having a ``tools.*`` configuration for defining how the internal Conan artifacts downloads are cached seems wrong, as it has nothing to do with the tools, it is something "core"
- The other is functional: there might be users that want to cache only the third-party things downloaded from the internet, but not their own Artifactory files. Or maybe the cleaning policy is different, keeping like forever the files downloaded from ``source()`` from the internet, while cleaning from time to time the cache from internal conan artifacts.

Reply:
I am not saying this is immutable, it is true that it is not super-intuitive at first sight. Just trying to explain why it is the way it is. In Conan 1.X is already similar, ``conan.conf`` is the core one, while profiles ``tools.*`` is the downloaded-from-internet one.

Reply:
I see, I was not aware of that, I guess that users demanded and use that functionality 👍